.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,140 @@
+name: 'Version Bump'
+on:
+  workflow_dispatch:
+    inputs:
+      release_line:
+        description: 'Release Line'
+        required: true
+        default: next
+        type: choice
+        options:
+        - next
+        - main
+  
+  # Run Weekly on Wednesday at 10:00 UTC time
+  schedule:
+    - cron: "0 10 * * 3"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  RELEASE_NAME: ''
+  HAS_CHANGES: ''
+  CURRENT_VERSION: ''
+  RELEASE_VERSION: ''
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout backstage'
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      # Beginning of yarn setup
+      - name: use node.js 14.x
```

Comment:
Let's use node 16 to be a bit more future proof :)

Reply:
Yeah, for sure! Done 👍